playground/tests/test_alive.py

```diff
@@ -0,0 +1,11 @@
+import pytest_container
+from .conftest import TW
+
+CONTAINER_IMAGES = [TW]
+
+
+# def test_etc_os_release_present(auto_container: pytest_container.container.ContainerData):
+#     assert auto_container.connection.file("/etc/os-release1").exists
+
+def test_alive(auto_container: pytest_container.container.ContainerData, host):
+    res = host.run_expect([0],f"curl localhost:8001",).stdout.strip()
```

Comment:
I guess this fails because the container's entrypoint/cmd didn't launch yet. You don't see this in manual runs, because you're generally not as fast as the machine, which starts your container and immediately `curl`s the port. My recommendation is to use `HEALTHCHECK` to ensure that the application inside the container is up and running.

Reply:
Ahh... that might be a problem. Unfortantily, HEALTHCHECK is not an OCI compatible instruction.  I'll look for another option.